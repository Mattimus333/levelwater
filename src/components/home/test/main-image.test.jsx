import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MainImage from '../main-image.jsx';

describe('main image', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<MainImage />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
