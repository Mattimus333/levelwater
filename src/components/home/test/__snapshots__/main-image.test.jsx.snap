exports[`main image renders a snapshot 1`] = `
<div
  id="main-image" />
`;
